.github/workflows/python-publish.yml

```diff
@@ -14,19 +14,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
-        python-version: '3.x'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install twine build
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python -m build
-        twine upload dist/*
+        python-version: "3.10"
+    - name: Build and publish to pypi
+      uses: JRubics/poetry-publish@v1.11
+      with:
+        repository_username: ${{ secrets.PYPI_USERNAME }}
```

Comment:
I don't have permission to settings/secrets. Can you grant me access? ⭐

Reply:
Done